app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -893,15 +893,15 @@ class StateFragmentTest {
   }
 
   @Test
-  fun testStateFragment_inputRatio_submit_correctAnswerDisplayed() {
+  fun testStateFragment_inputRatio_correctAnswerSubmitted_correctAnswerDisplayed() {
```

Comment:
`testStateFragment_inputRatio_correctAnswerSubmitted_correctAnswerIsDisplayed`

nit change as we usually do `IsDisplayed`,`IsChecked`,`IsVisible` etc

Reply:
Done